tests/integration_tests/functional/test_drive_virtio.py

```diff
@@ -23,7 +23,7 @@ def partuuid_and_disk_path_tmpfs(rootfs_ubuntu_22, tmp_path):
     disk_path.unlink()
 
 
-def test_rescan_file(test_microvm_with_api):
+def test_rescan_file(test_microvm_with_api, io_engine):
```

Comment:
can we extend patch-related tests in https://github.com/firecracker-microvm/firecracker/blob/main/tests/integration_tests/functional/test_api.py with async too?

Reply:
done!